.github/workflows/main.yaml

```diff
@@ -1,33 +1,47 @@
-name: Push
+name: CI
 
 on:
   pull_request:
     branches:
       - main
   push:
+    branches:
+      - main
 
 concurrency:
-    group: ${{ github.ref == 'refs/heads/main' && format('ignore-main-{0}', github.run_id) || format('{0}-{1}', github.workflow, github.ref_name) }}
-    cancel-in-progress: true
+  group: ${{ github.ref == 'refs/heads/main' && format('ignore-main-{0}', github.run_id) || format('{0}-{1}', github.workflow, github.ref_name) }}
+  cancel-in-progress: true
 
 jobs:
   lint-report:
     uses: ./.github/workflows/lint-report.yaml
 
   static-analysis:
+    name: Static analysis
```

Comment:
We will need to update the branch protection rules to match this new name. (See checks status)